template/src/ontology/Makefile.jinja2

```diff
@@ -251,18 +251,30 @@ prepare_release: all_odk
 	rm -f $(CLEANFILES) &&\
 	echo "Release files are now in $(RELEASEDIR) - now you should commit, push and make a release \
         on your git hosting site such as GitHub or GitLab"
+	{% if project.make_diff %}$(MAKE) $(REPORTDIR)/release-diff.md{% endif %}
```

Comment:
Can we instead of this line here get a `.PHONY` goal around row 92, like:

```
{% if project.make_diff %}
release_diff: $(REPORTDIR)/release-diff.md
{% endif %}
```
and add `{% if project.make_diff %} release_diff{% endif %}` to the `all_odk` goal?

Reply:
ok